docs/features/configExport.md

```diff
@@ -0,0 +1,55 @@
+# Exporting configurations
+
+The plugin supports exporting existing configurations as YAML.
+This is a feature available to the Jenkins administrators under the `http://[your_jenkins_url]/configuration-as-code/` URL.
+
+Export feature is **NOT** intended to offer a directly usable jenkins.yaml configuration. 
```

Comment:
Sooo, I don't think this is accurate, and given we're writing documentation for this, I think it would be a good time to update the UI warning as well?

Anywhere where it isn't usable is a bug and in general it should be usable.

Reply:
But if it is possible to find bugs, some kind of disclaimer would be accurate. Maybe it only has to be re-phrased it, but I agree with @oleg-nenashev  and the current situation is that the yaml file might not be 100% usable in all cases.